protocol/examples/answer/general_message.json

```diff
@@ -0,0 +1,20 @@
+{
+    "jsonrpc": "2.0",
+    "id": 999,
+    "result": [
+        {
+            "data": "XXX",
+            "sender": "XXX",
+            "signature": "XXX",
+            "message_id": "XXX",
+            "witness_signatures": []
+        },
+        {
+            "data": "XXX",
+            "sender": "XXX",
+            "signature": "XXX",
+            "message_id": "XXX",
+            "witness_signatures": []
```

Comment:
this is invalid and will be rejected in our implementation.
the data needs to be base64url encoded and represent valid keys, hashes, etc.
if we want this to be usable for tests, we need it to be correct.

that's also why I suggested using examples generated from the application :-)